src/main/java/net/dries007/tfc/objects/blocks/agriculture/BlockCropSpreading.java

```diff
@@ -6,44 +6,34 @@
 package net.dries007.tfc.objects.blocks.agriculture;
```

Comment:
What's the plan for spreading crops? I thought the decision was to remove / replace them for the time being as their growth method heavily relies on randomticks?

Reply:
They are defunct. This code is basically useless. I left it in for the time being until something is done with it.